addons/van_jsx/src/index.d.ts

```diff
@@ -0,0 +1,13 @@
+import { State, StateView } from "vanjs-core";
+
+export interface ReadonlyState<T> extends StateView<T> {
```

Comment:
ooc, do you need the interface `ReadonlyState<T>` here? I think `StateView<T>` already has the `readonly` `val` property.

Reply:
should we rename createState to createRef ? but createRef is same as react legacy api createRef